packages/apps/src/app.ts

```diff
@@ -71,6 +71,12 @@ export type AppOptions = Partial<Credentials> & {
    * Activity Options
    */
   readonly activity?: AppActivityOptions;
+
+  /**
+   * The default connection name to use for the app
+   * @default 'graph'
+   */
+  readonly defaultConnectionName?: string;
```

Comment:
we should update the existing oauth settings type here https://github.com/microsoft/spark.js/blob/main/packages/apps/src/oauth.ts#L1 with a `default` or something, since we will most likely add auth settings as time goes on and this will keep a nice area for them all to live